### ISSUE TITLE
Failing when generating genesis block for custom application

### ISSUE THREAD

**Issue by ishantiw** [LiskHQ/lisk-sdk#7636]
### Expected behavior

- `genesis-bloc:create` command should work and should create valid `genesis_block.blob`

### Actual behavior

When running `genesis-bloc:create` command, it throws error `Error: Stored total supply is non zero but cannot be computed.`

### Steps to reproduce

- Run `./bin/run init`
- Run `./bin/run genesis-block:create` under the created application

### Which version(s) does this affect? (Environment, OS, etc...)

`6.0.0-alpha.4`

**Comment by shuse2** [comment]
This is resolved by #7572

**Comment by lavrd** [comment]
I am facing the same error.

How I generated genesis block.

```sh
docker run --rm -it -v "${PWD}/genesis:/root/genesis" --entrypoint="" lisk-helper \
    lisk-core genesis-block:create \
    -f /usr/local/share/.config/yarn/global/node_modules/lisk-core/config/devnet/genesis_assets.json \
    -o /root/genesis \
    -c /usr/local/share/.config/yarn/global/node_modules/lisk-core/config/devnet/config.json \
    --network devnet
```

```js
import { Application } from "lisk-sdk";
(async () => {
  const { app } = Application.defaultApplication({
    system: {
      logLevel: "trace",
    },
    genesis: {
      block: {
        fromFile: "/root/genesis/genesis_block.blob"
      },
      chainID: "12312312",
    }
  }, false);
  try {
    await app.run();
    console.log("app started");
  } catch (e) {
    console.error("failed to start app:", e);
    process.exit(1);
  }
})()
```

- "lisk-sdk": "6.0.0-beta.6"
- lisk-core@4.0.0-beta.4
- node:18
